README.md

```diff
@@ -82,7 +82,7 @@ so you'll need to activate them by putting this in your `init.vim` file:
 ```lua
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
-  ensure_installed = "all"      -- one of "all", "language", or a list of languages
+  ensure_installed = "all",     -- one of "all", "language", or a list of languages
```

Comment:
Not related to this PR ?

Reply:
so what? but related to setting up this plugin for Windows in a manner that's working.

Reply:
I' really like separating PRs by their meaning rather than `By the way I changed that too.`

Reply:
I won't open a PR for a comma. If this change is already in master a rebase would remove this.

Reply:
Let's keep it given the size of this change, but the fact that the only change is this file is a comma bothers me.

Reply:
Ok, I create a PR. Then, rebasing makes this disappear.